Microsoft.Toolkit.Uwp.UI.Controls.Input/RangeSelector/RangeSelector.Input.Drag.cs

```diff
@@ -0,0 +1,137 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// RangeSelector is a "double slider" control for range values.
+    /// </summary>
+    public partial class RangeSelector : Control
+    {
+        /// <summary>
+        /// Event raised when lower or upper range thumbs start being dragged.
+        /// </summary>
+        public event DragStartedEventHandler ThumbDragStarted;
+
+        /// <summary>
+        /// Event raised when lower or upper range thumbs end being dragged.
+        /// </summary>
+        public event DragCompletedEventHandler ThumbDragCompleted;
+
+        private void OnThumbDragStarted(DragStartedEventArgs e)
+        {
+            ThumbDragStarted?.Invoke(this, e);
+        }
+
+        private void OnThumbDragCompleted(DragCompletedEventArgs e)
+        {
+            ThumbDragCompleted?.Invoke(this, e);
+        }
```

Comment:
I found it weird these also weren't in the events file.

If you look at my [refactor branch](https://github.com/michael-hawker/UWPCommunityToolkit/tree/rangeselector-refactor) I also updated to the proper 'On___' pattern as otherwise these methods aren't that useful.

Reply:
How's it now?

Reply:
Weird, I couldn't reply before... added a new comment, missed making them `virtual` as well.